testcontainers-concord-core/src/main/java/ca/ibodrov/concord/testcontainers/Projects.java

```diff
@@ -31,12 +31,26 @@ public Projects(ApiClient apiClient) {
     }
 
     public ProjectOperationResponse create(String orgName, String projectName) throws ApiException {
+        return create(orgName, projectName, true); // true to maintain backward compatibility
+    }
+
+    public ProjectOperationResponse create(String orgName, String projectName, boolean allowPayloads) throws ApiException {
         ProjectEntry projectEntry = new ProjectEntry()
                 .name(projectName)
                 .orgName(orgName)
                 .acceptsRawPayload(Boolean.TRUE)
+                .rawPayloadMode(ProjectEntry.RawPayloadModeEnum.TEAM_MEMBERS)
                 .visibility(ProjectEntry.VisibilityEnum.PUBLIC);
 
-        return projectApi.createOrUpdateProject(orgName, projectEntry);
+        var project = projectApi.createOrUpdateProject(orgName, projectEntry);
+
+        if (allowPayloads) {
```

Comment:
Just curious, why not set `RawPayloadModeEnum.EVERYONE`?

Reply:
My rationale: when creating a project in <`2.25.0`, the default raw payload setting if you only include the "enabled" option, but no level, results in a project that allows payloads from the owner (or an admin). So I was trying to keep that effectively the same.

Unfortunately, doing that involves an extra change -> adding a team to the project, which was not a previous behavior of either this library or the Project API itself.

So something's going to change a little bit either way (`EVERYONE` vs `TEAM`). I think I just convinced myself `EVERYONE` is less of a change even though it's more permissive. If someone has a test that's sensitive to this change, they'd probably hit more unexpectedness from a `assumed team+access level` change than just the access level.